src/protocols/telnet/clipboard.c

```diff
@@ -56,6 +56,7 @@ int guac_telnet_clipboard_blob_handler(guac_user* user, guac_stream* stream,
 int guac_telnet_clipboard_end_handler(guac_user* user, guac_stream* stream) {
 
     /* Nothing to do - clipboard is implemented within client */
```

Comment:
Is this docstring now out of date?

Reply:
Yes, but I want to keep changes to pre-existing minimal, to minimise chances of merge issues. 

Unless you're keen, I propose we leave it as it is.